source/portable/os/ota_os_freertos.c

```diff
@@ -312,10 +331,14 @@ OtaErr_t OtaStopTimer_FreeRTOS( OtaTimerId_t otaTimerId )
     return otaErrRet;
 }
 
-OtaErr_t ota_DeleteTimer( OtaTimerId_t otaTimerId )
+OtaErr_t OtaDeleteTimer_FreeRTOS( OtaTimerId_t otaTimerId )
 {
+    configASSERT( ( otaTimerId >= OtaRequestTimer ) && ( otaTimerId < OtaNumOfTimers ) );
+
     OtaErr_t otaErrRet = OTA_ERR_UNINITIALIZED;
 
+    BaseType_t retVal = pdFALSE;
+
     if( otaTimer[ otaTimerId ] != NULL )
     {
         /* Stop the timer. */
```

Comment:
Should update the comment to say 'Delete the timer'

Reply:
Thanks , updating it

Reply:
Addressed in https://github.com/aws/ota-for-aws-iot-embedded-sdk/pull/91/commits/935a2ddcdf079e6aca118c6dd4266df35a825df0